README.md

```diff
@@ -49,6 +49,12 @@ Detailed documentation for configuring GitOps/ArgoCD can be found in [`GITOPS-CO
 
 Detailed documentation for configuring Pipelines/Tekton can be found in [`PIPELINES-CONFIG.md`](./docs/PIPELINES-CONFIG.md)
 
+## Developer Hub Configuration
```

Comment:
I think we need to either

1. emphasize more on running these configuration scripts or,
2. streamline better through a main script like configure.sh

Otherwise, users will encounter errors when trying to login to RHDH via GitHub login.

I am trying to compare the UX between the rhtap-installer and this installer. rhtap-installer had two steps:

1. helm install/bash script calling helm install
2. running a PLR that was output from previous step 1 and it gives you callback url, webhook url, webhook secret, etc.

Reply:
> running a PLR that was output from previous step 1 and it gives you callback url, webhook url, webhook secret, etc.

@maysunfaisal  Done in https://github.com/redhat-ai-dev/ai-rhdh-installer/pull/10/commits/3a820df5e477b1d8c80c23fb8bdd2951e9527afe

Reply:
> streamline better through a main script like configure.sh

@maysunfaisal Provided in https://github.com/redhat-ai-dev/ai-rhdh-installer/pull/10/commits/4aec9db72f787370738cb19095c0e089e447906a

Reply:
@maysunfaisal Is there any other UX improvements that should be included in these changes?

Reply:
it looks good now, thx